.sage/sagefile.go

```diff
@@ -73,3 +73,9 @@ func GitVerifyNoDiff(ctx context.Context) error {
 	sg.Logger(ctx).Println("verifying that git has no diff...")
 	return sggit.VerifyNoDiff(ctx)
 }
+
+func GoIamctl(ctx context.Context) error {
+	sg.Logger(ctx).Println("installing iamctl...")
+	bashCommand := "cd ./cmd/iamctl && go install ."
+	return sg.Command(ctx, "bash", "-c", bashCommand).Run()
```

Comment:
Bash is not needed for this, you can just do something like 
```
cmd := sg.Command(ctx, "go", "install", ".")
cmd.Dir = sg.FromGitRoot("cmd/iamctl")
cmd.Run()
```

Reply:
Thanks, updated this